torch/_inductor/kernel/flex_attention.py

```diff
@@ -871,14 +890,36 @@ def lower_cpu(
         )
 
     fake_buffers: List[Buffer] = []  # noqa: F821
+
+    # The value of cur_qSplitSize and cur_kvSplitSize are decided during runtime.
```

Comment:
Maybe ahve one named '[Note]' and point this comment there

Reply:
I've added a `[Note] Handle the case where the split sizes are not statically known` comment and inside `CppFlexAttentionTemplate` where we replace the string, I pointed this comment to make it clearer.